meta-balena-common/classes/kernel-balena.bbclass

```diff
@@ -217,6 +217,11 @@ BALENA_CONFIGS[firmware_compress] = " \
     CONFIG_FW_LOADER_COMPRESS=y \
 "
 
+KERNEL_ZSTD = "${@configure_from_version("5.9", "kernel_zstd", "", d)}"
```

Comment:
@jakogut isn't the idea to use zstd if the kernel support it? If so, shouldn't it be added to the `BALENA_CONFIGS` variable?
```
BALENA_CONFIGS ?= " \
     dmcrypt \
     no_gcc_plugins \
     ${FIRMWARE_COMPRESS} \
+   ${KERNEL_ZSTD} \
     "
```

Reply:
Yep, good spot